bin/docker/kafka_bridge_config_generator.sh

```diff
@@ -122,6 +122,7 @@ HTTP_PROPERTIES=$(cat <<-EOF
 http.enabled=${KAFKA_BRIDGE_HTTP_ENABLED}
 http.host=${KAFKA_BRIDGE_HTTP_HOST}
 http.port=${KAFKA_BRIDGE_HTTP_PORT}
+http.enable_cors=${KAFKA_BRIDGE_§CORS_ENABLED}
```

Comment:
I guess the `§` in `KAFKA_BRIDGE_§CORS_ENABLED` is a typo?

Reply:
correct! I will fix it